### PR TITLE
Implement windows port for castanets.

### PR DESCRIPTION
**Pull request by suyambulingamrm**
This CL is initial changes required for basic rendering in
windows port.

Signed-off-by: suyambu.rm <suyambu.rm@samsung.com>

### CODE REVIEW

File: mojo/edk/system/broker_win.cc [hunk 1]
```diff
@@ -156,5 +180,5 @@ scoped_refptr<PlatformSharedBuffer> Broker::GetSharedBuffer(size_t num_bytes) {
   return nullptr;
 }
 
-}  // namespace edk
+}  // namespace mojo
```
Comment (Sunwoo-Nam) [inline mojo/edk/system/broker_win.cc]: It seems namespace mojo is closed two times. Is there no compile error?

File: mojo/edk/system/broker_win.cc [hunk 2]
```diff
@@ -85,10 +103,10 @@ Channel::MessagePtr WaitForBrokerMessage(PlatformHandle platform_handle,
   }
 
   return message;
-}
-
 }  // namespace
 
+}  // namespace edk
+
```
Comment (suyambulingamrm) [inline mojo/edk/system/broker_win.cc]: Empty namespace is closed here.

File: mojo/edk/system/broker_win.cc (hunk 1 above)
Comment (suyambulingamrm) [reply mojo/edk/system/broker_win.cc]: There are three namespaces in the file. 1. mojo, 2. edk. 3. a empty name space.
3 is closed above as I mentioned.
In this line edk and mojo namespaces are closed as usual.
Some how Visual studio is integrated with the clang-format. Once I close a file auto format is done.
That is the reason many files have formatting changes also.

**Comment by Sunwoo-Nam** [comment]
You have already commented about automatic clang formatting.. Sorry I missed it.
I agree with your solution removing files.

File: mojo/edk/system/broker_win.cc (hunk 1 above)
Comment (suyambulingamrm) [reply mojo/edk/system/broker_win.cc]: Corrected on latest patch set.

File: content/browser/renderer_host/media/audio_sync_reader.cc [hunk 3]
```diff
@@ -187,11 +190,15 @@ void AudioSyncReader::RequestMoreData(base::TimeDelta delay,
   }
 
 #if defined(CASTANETS)
-  size_t sent_bytes = HANDLE_EINTR(send(accept_handle_.get().handle, &control_signal, sizeof(control_signal), MSG_NOSIGNAL));
+  size_t sent_bytes = 0;
+#if !defined(OS_WIN)
+  HANDLE_EINTR(send(accept_handle_.get().handle, &control_signal,
```
Comment (Sunwoo-Nam) [inline content/browser/renderer_host/media/audio_sync_reader.cc]: I think it is better to put the return value of send() in 'sent_bytes'.
sent_bytes = HANDLE_EINTR(send(accept_handle_.get().handle, ...

Comment (suyambulingamrm) [reply content/browser/renderer_host/media/audio_sync_reader.cc]: Yes, its a mistake. Thanks for pointing.

**Pull request merged**